Samples/UIWidgetsSamples_2019_4/Assets/UIWidgetsGallery/GalleryMain.cs

```diff
@@ -8,6 +9,14 @@ public class GalleryMain : UIWidgetsPanel {
             ui_.runApp(new GalleryApp());
         }
 
+        protected override void loadConfiguration()
```

Comment:
I really prefer to name this function as onEnable() :-)

Reply:
OK， I‘ll fix it.